CharacterGenerator/CharacterGenerator/Bender.m

```diff
@@ -0,0 +1,89 @@
+//
+//  Bender.m
+//  CharacterGenerator
+//
+//  Created by Christian Maldonado on 8/4/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "Bender.h"
+#import "BenderManager.h"
+#import "CharacterTableViewController.h"
+
+@implementation Bender
+
+-(void)animalForColor:(NSString *)color AndTexture:(NSString *)texture{
+    
+    
+    [self colorsAndTexturesToNumbers];
+    
+    NSString *c = [self.colorsToNumbers objectForKey:color];
+    NSString *t = [self.texturesToNumbers objectForKey:texture];
+    
+    NSInteger colorValue = [c integerValue];
+    NSInteger textureValue = [t integerValue];
+    
+    NSInteger value = colorValue * textureValue;
+    //Scorpion
+    if(value == 22 || value == 85 || value == 91 || value == 133){
```

Comment:
Add spaces to improve readability, i.e. `if (value == 22 || value == 85 || value == 91 || value == 133) {`